tests/test_fmu_dataio_generic.py

```diff
@@ -72,10 +76,67 @@ def test_get_meta_masterdata():
 
 def test_get_meta_access():
     """The private routine that provides access."""
+
+    # test case 1: No config
+    case = fmu.dataio.ExportData()
+    case._get_meta_access()
+    assert case.metadata4access == {}  # detecting empty dict
+
```

Comment:
Perhaps it should fail before `case.metadata4access` becomes empty

Reply:
Perhaps related to the previous comment on missing config. If we do not allow missing config, then this test becomes obsolete and would be removed. (Or replaced by a more targeted test testing for missing config.)